content/docs/sidebar.json

```diff
@@ -35,36 +35,49 @@
     },
     "children": [
       {
-        "slug": "data-and-model-versioning",
-        "tutorials": {
-          "katacoda": "https://katacoda.com/dvc/courses/get-started/versioning"
-        }
-      },
-      {
-        "slug": "data-and-model-access",
-        "tutorials": {
-          "katacoda": "https://katacoda.com/dvc/courses/get-started/accessing"
-        }
-      },
-      {
-        "slug": "data-pipelines",
-        "tutorials": {
-          "katacoda": "https://katacoda.com/dvc/courses/get-started/stages"
-        }
-      },
-      {
-        "label": "Metrics, Parameters, and Plots",
-        "slug": "metrics-parameters-plots",
-        "tutorials": {
-          "katacoda": "https://katacoda.com/dvc/courses/get-started/params-metrics-plots"
-        }
+        "slug": "data",
+        "label": "Data Management",
+        "source": "data/index.md",
+        "children": [
```

Comment:
The only issue with adding this middle click-level is that it's possible to "get lost" in the navigation. Example: I click Get Started and read the general index. Then I only open the Data Management element in the nav (click the little arrow) and see that there's pages inside. I then assume the content starts in the first sub-page (Accessing Data) and click on that. I have now missed all of the basic info on data mgmt. Cc @shcheklein 

![image](https://user-images.githubusercontent.com/1477535/165393264-a7bad7f0-1f45-4454-a7fe-528785dd0414.png)